tests/integration/transactions/transactions.spec.ts

```diff
@@ -45,15 +46,15 @@ describe(`Transactions integration tests`, () => {
     );
     expect(buttonText).toContain('Contract call');
   });
-
-  it('validates against insufficient funds when performing a contract call', async () => {
-    const [popup] = await Promise.all([
-      browser.context.waitForEvent('page'),
-      demo.page.click(createTestSelector(TransactionsSelectors.BtnContractCall)),
-    ]);
-    const errorMsg = await popup.textContent(
-      createTestSelector(TransactionsSelectors.TransactionErrorMessage)
-    );
-    expect(errorMsg).toBeTruthy();
-  });
+  //
```

Comment:
Is this not working?

Reply:
Due to the reset yesterday of testnet it was failing

Reply:
seems to be working now!